src/types/opt_arg.rs

```diff
@@ -0,0 +1,55 @@
+use std::fmt;
+
+use crate::NativeRepr;
+
+/// A convenience type to explicitly mark
+/// a function argument as `opt T`.
+///
+/// > When left unspecified on Redscript side,
```

Comment:
```suggestion
/// When left unspecified on Redscript side,
```
is the > intentional?

Reply:
Yeah, but it can as well be removed.

Reply:
just not sure what it does

Reply:
an indented comment, I removed it.